allensdk/brain_observatory/ecephys/stimulus_table/output_validation.py

```diff
@@ -13,8 +13,8 @@ def validate_epoch_durations(table, start_key="Start", end_key="End"):
             UserWarning,
         )
     if min_duration < 0:
-        print(table.loc[min_duration_index - 5 : min_duration_index + 5, :])
-        raise ValueError(f"there is an epoch with negative duration (index: {min_duration_index}")
+        # print(table.loc[min_duration_index - 5 : min_duration_index + 5, :])
+        warnings.warn(f"there is an epoch with negative duration (index: {min_duration_index}")
```

Comment:
can you make this enforcement configurable? I am worried about just leaving this in place for future sessions.

Reply:
What should be the default behavior?